tests/TestCli.cpp

```diff
@@ -224,9 +228,6 @@ void TestCli::testAdd()
                     "newuser2",
                     "--url",
                     "https://example.net/",
-                    "-g",
```

Comment:
@droidmonkey `-g` and `-p` options were used at the same time, and the password prompt was winning. This case is now explicitly handled by both `add` and `edit`.